### PR TITLE
[zh-cn] Update replicaset.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -239,6 +239,8 @@ Take the previous frontend ReplicaSet example, and the Pods specified in the fol
 ReplicaSet 的选择算符相匹配的标签。原因在于 ReplicaSet 并不仅限于拥有在其模板中设置的
 Pod，它还可以像前面小节中所描述的那样获得其他 Pod。
 
+以前面的 frontend ReplicaSet 为例，并在以下清单中指定 Pods:
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:242]:
```suggestion
以前面的 frontend ReplicaSet 为例，并在以下清单中指定这些 Pod：
```

**Comment by windsonsea** [comment]
This file may has more inconsistencies with en. It's not encouraged to submit part of changes.

You can run './scripts/lsync.sh' or 'vimdiff' to find those differences:

image

**Comment by gaogao101** [comment]
@windsonsea Thanks for your help. The modification has been completed.

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -460,8 +462,8 @@ When using the REST API or the `client-go` library, you must set `propagationPol
 ```shell
 kubectl proxy --port=8080
 curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
-  -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
-  -H "Content-Type: application/json"
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
+> -H "Content-Type: application/json"
```
Comment (chengxiangdong) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:466]: It seems that `>` has changed its meaning.


Before change
```shell
kubectl proxy --port=8080
curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
  -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
  -H "Content-Type: application/json"
```

After change
```shell
kubectl proxy --port=8080
curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
> -H "Content-Type: application/json"
```

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 3]
```diff
@@ -485,8 +487,8 @@ For example:
 ```shell
 kubectl proxy --port=8080
 curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/frontend' \
-  -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Orphan"}' \
-  -H "Content-Type: application/json"
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Orphan"}' \
+> -H "Content-Type: application/json"
```
Comment (chengxiangdong) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:491]: ditto

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md (hunk 2 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:466]: right, the `>` character should be removed.

**Comment by gaogao101** [comment]
@chengxiangdong @tengqm Thank you for your help. The modification has been completed.

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 4]
```diff
@@ -118,7 +118,7 @@ And see the frontend one you created:
 -->
 并看到你所创建的前端：
 
-```
+```shell
 NAME       DESIRED   CURRENT   READY   AGE
 frontend   3         3         3       6s
 ```
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:124]: This is not a shell script. We (previous contributor) are intentionally remove the wrong syntax lang tag.

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 5]
```diff
@@ -239,6 +239,8 @@ Take the previous frontend ReplicaSet example, and the Pods specified in the fol
 ReplicaSet 的选择算符相匹配的标签。原因在于 ReplicaSet 并不仅限于拥有在其模板中设置的
 Pod，它还可以像前面小节中所描述的那样获得其他 Pod。
 
+以前面的 frontend ReplicaSet 为例，并在以下清单中指定这些 Pod:
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:242]:
```suggestion
以前面的 frontend ReplicaSet 为例，并在以下清单中指定这些 Pod：
```

File: content/zh-cn/docs/concepts/workloads/controllers/replicaset.md [hunk 6]
```diff
@@ -179,7 +179,7 @@ You should see Pod information similar to:
 -->
 你会看到类似如下的 Pod 信息：
 
-```
+```shell
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/replicaset.md:182]: 这个无意义。或者去掉 shell，或者将其改为 none

**Pull request merged**